spec/features/auth_spec.rb

```diff
@@ -16,4 +16,11 @@
     expect(page).to have_content("user@example.com")
   end
 
+
+  scenario "Users can register" do
+    visit root_path
```

Comment:
In here you should be filling in a form before clicking Register, then asserting that the user sees what a logged in user sees